google/resource_container_cluster.go

```diff
@@ -236,10 +236,9 @@ func resourceContainerCluster() *schema.Resource {
 				ForceNew: true, // TODO(danawillow): Add ability to add/remove nodePools
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
-						"initial_node_count": {
```

Comment:
Should we keep the old field and marked it as deprecated or removed? this way, we can show an error message to the user